developer-workflow/sbom.rst

```diff
@@ -107,3 +107,19 @@ When removing a dependency:
    that correct package is removed from the SBOM.
 5. Commit the changes to :cpy-file:`Misc/sbom.spdx.json` and
    :cpy-file:`Tools/build/generate_sbom.py`.
+
+Updating external dependencies (cpython-source-deps)
+----------------------------------------------------
+
+Dependencies for Windows CPython builds are `stored in a separate repository <https://github.com/python/cpython-source-deps>`_
+and then fetched during builds of CPython for Windows in the script :cpy-file:`PCbuild/get_externals.bat`.
+
+In this script the libraries to fetch are designated by ``{name}-{version}`` git refs being added to the ``libraries`` variable.
+SBOM tooling in the CPython repository matches these git refs in order to build the :cpy-file:`Misc/externals.spdx.json`
+SBOM file. When updating external dependencies for a CPython branch:
+
+1. Push the update to the ``cpython-source-deps`` repository and create a new git tag.
```

Comment:
Maybe worth noting that this can only be done by a core committer, and we don't accept PRs for it (because we need to verify the sources have come from the right source and are unmodified, and our trust boundary for this is "has the commit bit").

Also might be worth noting that sometimes there's a build step involved and the core committer will then push a tag to `cpython-bin-deps` that will actually be used in the build. Tcl/Tk, libffi and OpenSSL are all in this group.

In practice for contributors, what this usually means is that they should post an issue requesting the updated version, wait for a core dev to say the tags are ready, and then the contributor can continue with the following steps.

Reply:
I've addressed this comment in https://github.com/python/devguide/pull/1280/commits/b32b691f14434299aa29b054061a139d77dea8f0. Do you think we should cover the cpython-bin-deps part here as well?

Reply:
Not in the same note, but it ought to be documented somewhere. At the very least, we should mention the `cpython-bin-deps` repo at least once so that someone reading this knows to look there.